src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.spec.ts

```diff
@@ -126,6 +132,50 @@ describe('PaypalCommerceButtonStrategy', () => {
         expect(paypal.Buttons).toHaveBeenCalledWith({
             createOrder: expect.any(Function),
             onApprove: expect.any(Function),
+            style: paypalOptions.style,
+        });
+    });
+
+    it('validate style for PaypalCommerce checkout button', async () => {
```

Comment:
it's better to be more descriptive for the unit tests, for example
`it calls paypal API with the provided styles`

Reply:
Thank you!